flight/goto/mission/main.py

```diff
@@ -11,8 +11,20 @@
 import math
 import typing
 from typing import Dict,List
+import sys
 
 async def run() -> None:
+    """
+    This function is just a driver to test the goto function.
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    None
+    """
```

Comment:
This one is on me here, I told you to make a docstring however since this function doesn't have arguments and returns nothing, it doesn't need these sections.